introspection-engine/connectors/sql-introspection-connector/src/calculate_datamodel.rs

```diff
@@ -85,6 +333,20 @@ fn parse_int(value: &str) -> Option<i32> {
     }
 }
 
+fn parse_bool(value: &str) -> Option<bool> {
+    debug!("Parsing bool '{}'", value);
+
+    //    if value.to_lowercase() == "true".to_string() {
+    //        Some(1)
+    //    } else if value.to_lowercase() == "false".to_string() {
+    //        Some(0)
+    //    } else {
+    //        None
+    //    }
+    //
+    value.parse().ok()
```

Comment:
I think we have to lowercase the value first, the standard lib only parses exactly "true" and "false".

I checked on the playground: https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=70f15aaaea1eb42194a7285c8298a067

Reply:
Done